docker/frontend-entrypoint.py

```diff
@@ -11,6 +11,7 @@ def main():
     public = os.environ.get('REACT_APP_PUBLIC_GRAPH_URI')
     frontend = os.environ.get('REACT_APP_FRONTEND_URI')
     auth = os.environ.get('REACT_APP_AUTH_MODE')
+    otel = os.environ.get('REACT_APP_OTLP_ENDPOINT')
```

Comment:
you may want to check the generated `constants.js` file to make sure this isn't being elided

Reply:
Confirmed it's in there! Here's what I see in a local build:

```ts
var _ = { BASE_URL: "/", MODE: "production", DEV: !1, PROD: !0, SSR: !1 };
const o = _.REACT_APP_PRIVATE_GRAPH_URI ?? window.location.origin + "/private",
  t = _.REACT_APP_PUBLIC_GRAPH_URI || "https://pub.highlight.io",
  P =
    _.REACT_APP_FRONTEND_URI ||
    window.location.protocol + "//" + window.location.host,
  R = _.REACT_APP_AUTH_MODE,
  A = _.REACT_APP_OTLP_ENDPOINT || "https://otel.highlight.io:4318";
export { R as A, P as F, A as O, o as P, t as a };
```